code/modules/power/singularity/generator.dm

```diff
@@ -10,6 +10,16 @@
 	var/energy = 0
 	var/creation_type = /obj/singularity
 
+	// You can buckle someone to the singularity generator, then start the engine. Fun!
+ 	can_buckle = 1
+ 	buckle_lying = 0
+ 	buckle_requires_restraints = 1

```

Comment:
This means you can't buckle someone here unless they're cuffed. Intended?

Reply:
ah yes